scripts/jenkins/Jenkinsfile

```diff
@@ -741,36 +741,64 @@ def buildDoc() {
           [parserName: 'Doxygen', pattern: 'build/doc/doxygen.log']
         ]
 
-        // TODO don't write to latest on PR's
-        sshPublisher(
-          publishers: [
-            sshPublisherDesc(
-              configName: 'doc.libelektra.org',
-              transfers: [
-                sshTransfer(
-                  sourceFiles: 'build/doc/latex/*',
-                  removePrefix: 'build/doc/',
-                  remoteDirectory: 'api/latest'
-                ),
-                sshTransfer(
-                  sourceFiles: 'build/doc/man/*',
-                  removePrefix: 'build/doc/',
-                  remoteDirectory: 'api/latest'
-                ),
-                sshTransfer(
-                  sourceFiles: 'doc/help/*.html',
-                  removePrefix: 'doc/help/',
-                  remoteDirectory: 'help'
-                ),
-                sshTransfer(
-                  sourceFiles: 'build/API_DOC/*',
-                  removePrefix: 'build/API_DOC/'
-                )
-              ]
-            )
-          ],
-          verbose: true
-        )
+        if (isMaster()) {
```

Comment:
The if- and else-block look very similar:

<img width="953" alt="compare" src="https://user-images.githubusercontent.com/691989/58787135-b05fa880-85d8-11e9-8395-ee084d87e4d6.png">

. It might make sense to set a variable to the value `lastest` or `pr` and remove the `if`-statement altogether.

Reply:
Yes, it makes sense to also build the API_DOC for PRs (once they are fixed), then the if can be removed.